TodoApp/src/TodoApp.Api/App_Start/RouteConfig.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Web.Http;
+using System.Web.Http.Routing;
+using Microsoft.Web.Http.Routing;
+
+namespace TodoApp.Api
+{
+    public static class RouteConfig
+    {
+        public static void Register(HttpConfiguration config)
+        {
+            // Web API routes
+            var versionConstraintResolver = new DefaultInlineConstraintResolver
+            {
+                ConstraintMap =
+                    {["apiVersion"] = typeof(ApiVersionRouteConstraint)}
+            };
+
+            config.MapHttpAttributeRoutes(versionConstraintResolver);
+
+            config.Routes.MapHttpRoute(
+                "DefaultApi",
```

Comment:
You need this string to match to [that string](https://github.com/LindaLangerova/kentico-onboarding-cs/pull/3/files#diff-ac8c2c8b1dd3289e49a824ad7f5c251eR19) in `UrlGenerator` – always… What a constant phenomenon 😆